dialog_lib/db/utils.py

```diff
@@ -1,9 +1,9 @@
 import uuid
-
+from .session import get_session
 from .models import Chat
 
 
-def create_chat_session(identifier=None, dbsession=None, model=Chat):
+def create_chat_session(identifier=None, dbsession=get_session(), model=Chat):
```

Comment:
The `dbsession` argument now is a generator object, not a SQLAlchemy session

Reply:
Solved in the latest commit!